src/sentry/models/sentryapp.py

```diff
@@ -72,6 +78,16 @@ def track_response_code(status, integration_slug, webhook_event):
     )
 
 
+class SentryAppManager(BaseManager):
```

Comment:
I think maybe we want to use https://github.com/getsentry/sentry/blob/master/src/sentry/db/models/paranoia.py#L18 instead?

since the `SentryApp` table use the `ParanoidModel`